Penteract/Assets/Scripts/HUDController.h

```diff
@@ -117,6 +122,7 @@ class HUDController : public Script {
 
 	float remainingTimesFang[MAX_HEALTH] = { 0,0,0,0,0,0,0 };
 	float remainingTimesOni[MAX_HEALTH] = { 0,0,0,0,0,0,0 };
+	bool abilityCoolDownsRetreived[Cooldowns::TOTAL] = { true,true,true,true,true,true,true };
```

Comment:
I couldn't comment on the declaration of the Enum Cooldowns, but if it isn't used somewhere else, have it as private enum

Reply:
Changed to enum class on Jaime's orders, forward declaration of enum appears troublesome

Reply:
What I meant is to have the declaration of the enum inside the private scope, not outside the class.